cmd/obslytics/export.go

```diff
@@ -66,20 +67,47 @@ func registerExport(m map[string]setupFunc, app *kingpin.Application) {
 			if err != nil {
 				return err
 			}
-			out, err := outfactory.Parse(outputCfg)
+			out, err := outfactory.Parse(logger, outputCfg)
```

Comment:
I would see object storage as different output essentially (:

Reply:
changed it :+1: 
I wasn't sure if returning more than two items was good practice.